gpMgmt/bin/lib/gp_bash_functions.sh

```diff
@@ -171,13 +171,51 @@ WARN_MARK="<<<<<"
 # Functions
 #******************************************************************************
 
+#
+# Simplified version of _nl_normalize_codeset from glibc
+# https://sourceware.org/git/?p=glibc.git;a=blob;f=intl/l10nflist.c;h=078a450dfec21faf2d26dc5d0cb02158c1f23229;hb=HEAD
+#
+
+NORMALIZE_CODESET_IN_LOCALE () {
+	IFS=".|@" read -a arr_locale <<< $1
+	local len=${#arr_locale[@]}
+
+	if [[ $len == 1 ]];
+	then
+		echo $1
+	else
+		local normalized_locale=${arr_locale[0]}
+		local codeset=${arr_locale[1]}
+		local desc=${arr_locale[2]}
```

Comment:
the names of `normalized_locale` and `desc` variables are not descriptive about stored values